src/OscapScannerRemoteSsh.cpp

```diff
@@ -415,8 +415,11 @@ QString OscapScannerRemoteSsh::createRemoteTemporaryFile(bool cancelOnFailure)
                         "data to! Diagnostic info: %1").arg(proc.getDiagnosticInfo())
         );
 
-        mCancelRequested = true;
-        signalCompletion(mCancelRequested);
+        if (cancelOnFailure) {
```

Comment:
No callers set `cancelOnFailure=false` (so the change makes no difference), but I believe this is the correct way to implement this behavior. I'd appreciate comments if its not.